examples/tests/invalid/json.w

```diff
@@ -20,7 +20,10 @@ let forever_json = Json {a: "hello"};
 forever_json.set("a", "world!");
 //           ^^^ Unknown symbol "set" (TODO: better error message https://github.com/winglang/wing/issues/1660) 
 
-// Unallowed Json types TODO: https://github.com/winglang/wing/issues/1683
-// let bkt = new cloud.Bucket();
-// let j_arr = Json [bkt];
-//                   ^^^ type cloud.bucket is not serializable 
\ No newline at end of file
+let bkt = new cloud.Bucket();
+let j_arr = Json [bkt];
+//                ^^^ Expected "Json" elements to be serializable, but got "Bucket" which is not serializable
+
+let json_obj = Json { boom: bkt };
+//                ^^^ Expected "Json" elements to be serializable, but got "Bucket" which is not serializable
```

Comment:
The tics should be under `bkt`

Reply:
We do not support any serializable value in Json we only support str,num, bool and arrays with these values (or another json)  - https://docs.winglang.io/reference/spec#114-json-type

I would change the error message and remove serializable 

//                ^^^ Expected "Json" elements to be Json Value (https://www.json.org/json-en.html)